plugin/src/main/java/org/screamingsandals/bedwars/utils/MiscUtils.java

```diff
@@ -52,6 +52,7 @@
 
 @UtilityClass
 public class MiscUtils {
+    private final Random random = new Random();
```

Comment:
name should be uppercase ([Java naming conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html), bottom of the page), because the variable is implicitly static (due to UtilityClass annotation)